src/java/jni.rs

```diff
@@ -997,4 +999,132 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn check_gen_jni_function_namespace_and_cfg() {
+        let func_name = "testDummyFuncName";
+        let native_name = "test_dummy_func_name";
+        let mut ctx = Context::default();
+        let mut dummy_outputs: HashMap<String, String> = HashMap::new();
+        let dummy_func_str = indoc!(
+        "/// Comments are dumb, we don't need them here.
+        #[cfg(feature = \"mock-routing\")]
+        #[no_mangle]
+        pub unsafe extern \"C\" fn test_dummy_func_name(
+            dummy_app: *mut App,
+            dummy_user_data: *mut c_void,
+            dummy_o_cb: extern \"C\" fn(dummy_user_data: *mut c_void, dummy_result: *const FfiResult),
+        ) {}"
+        );
+        let test_str = "# [ cfg ( feature = \"mock-routing\" ) ]\n\
```

Comment:
This var could have a more descriptive name, like e.g.
```suggestion
        let expected_jni_code = "# [ cfg ( feature = \"mock-routing\" ) ]\n\
```